### ISSUE TITLE
TooltipConfig is not defined

### ISSUE THREAD

**Issue by MrWildanMD** [N00nDay/stwui#162]
i'm using stwui v18.0 then got the following error hence i'm not even using tooltip
```
logs:

Error: Parse failure: Export 'TooltipConfig' is not defined (7:18)
At file: /node_modules/.pnpm/stwui@0.18.0-next_@sveltejs+kit@1.25.1_svelte@4.2.1/node_modules/stwui/actions/index.js
Contents of line 7: export { tooltip, TooltipConfig, clickOutside, forwardEventsBuilder, useActions, SvelteActionReturnType, SvelteHTMLActionType, HTMLActionEntry, HTMLActionArray, SvelteSVGActionType, SVGActionEntry, SVGActionArray, ActionArray, clipboard, floatingUI };
```

**Comment by N00nDay** [comment]
Thank you. I will dig into it this morning and get this resolved. There must be a conflict between tippy.js and something else as I just updated some dependencies.

**Comment by N00nDay** [comment]
There is a problem with how STWUI is being packaged where types are not being stripped out as they were prior to v0.18.0-next. I am still investigating what caused this issue as I have reverted back to an old build but still have the same problems. It is advised to roll back to v0.17.4-next until this is resolved.

**Comment by MrWildanMD** [comment]
Understandable, thanks for your hardwork. I'd like to contribute if i had enough experience in svelte

**Comment by N00nDay** [comment]
V0.18.4-next is a temporary patch that fixes this issue. Still working on a permanent patch but it is usable for now.

**Comment by MrWildanMD** [comment]
Okay thank you, one more thing..can you also check for the Datepicker behaviour? when i wrap it inside form after picking date, it just like submit and appending the picked date to url..and i'm still figuring out what's wrong with my implementation so i cant wrap it inside a form

**Comment by N00nDay** [comment]
Makes total sense. I will open a separate issue. I know what the issue is and it should be relatively easy to resolve. Should be fixed today.

**Comment by N00nDay** [comment]
@MrWildanMD the DatePicker auto submission when wrapped in forms should be fixed with the latest update v0.18.4-next.
